src/modules/uORB/SubscriptionCallback.hpp

```diff
@@ -160,7 +160,7 @@ class SubscriptionCallbackWorkItem : public SubscriptionCallback
 	{
 		// schedule immediately if updated (queue depth or subscription interval)
 		if ((_required_updates == 0)
-		    || (_subscription.get_node()->published_message_count() >= (_subscription.get_last_generation() + _required_updates))) {
+		    || ((_subscription.get_node()->published_message_count() - _subscription.get_last_generation()) >= _required_updates)) {
```

Comment:
What's the plan here?

Reply:
Use subtraction to be compatible with wrap-around.
The previous addition will be wrong in the wrap-around situation.

Reply:
I think it's worth a try. Would something like `updates_available()` be a better name?

Reply:
Yes, in terms of function naming, I follow your opinions.